cmd/eval.go

```diff
@@ -35,14 +39,20 @@ func formatReplacements(replacements map[string]string) map[string][]string {
 	return formattedReplacements
 }
 
-// validateFile is a simple wrapper to ensure the input/output of valid YAML
-func validateFile(file []byte) error {
-	var holder interface{}
-	err := yaml.Unmarshal(file, &holder)
-	if holder == nil {
+// validateFile is a simple wrapper to ensure the file we're using exists, is readable,
+// and is valid YAML
+func validateFile(file io.Reader) error {
+	fileBytes, err := ioutil.ReadAll(file)
+	if err != nil {
+		return ErrCouldNotReadFile
```

Comment:
I am of the opinion that you should surface the error returned by the `ioutil.ReadAll` function, it could contain  critical debugging data. With this custom error, I only know that I `could not read file`. This error is also [declared in a separate package](https://github.com/operator-framework/combo/pull/38/files#diff-4c6785786c3eaa4a21b69e4a2a47830bb533b95d63b5b6c5bf69bc7e2d76625cR13), and when it is used [the error is surfaced](https://github.com/operator-framework/combo/pull/38/files#diff-4c6785786c3eaa4a21b69e4a2a47830bb533b95d63b5b6c5bf69bc7e2d76625cR27).